table/table_test.cc

```diff
@@ -8621,6 +8621,124 @@ TEST_P(UserDefinedIndexTest, ConfigTest) {
   ASSERT_OK(DestroyDB(dbname, options_));
 }
 
+TEST_P(UserDefinedIndexTest, RangeDelete) {
+  BlockBasedTableOptions table_options;
+  options_.num_levels = 50;
+  options_.compaction_style = kCompactionStyleUniversal;
```

Comment:
Can you set `options_.disable_auto_compactions`? Without it, the delete range file may just get compacted away